Podfile

```diff
@@ -5,7 +5,7 @@ inhibit_all_warnings!
 
 target 'HackIllinois' do
     use_frameworks!
-    pod 'lottie-ios', '~> 2.1.4'
+    pod 'lottie-ios'
```

Comment:
You should pin this to an appropriate version so we don't have developers with newer/older Lottie versions